commit 8b0acb452cf0a1bab81b5c3a90e088f9e8c57068 (refs/changes/40/4640/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-08-06T23:18:08+00:00 (1 year, 2 months ago)
    
    TRITON-668 Add SRV records to FWAPI
    Reviewed by: Nick Zivkovic <nick.zivkovic@joyent.com>
    Approved by: Nick Zivkovic <nick.zivkovic@joyent.com>

diff --git a/sapi_manifests/registrar/manifest.json b/sapi_manifests/registrar/manifest.json
new file mode 100644
index 0000000..98cd3bb
--- /dev/null
+++ b/sapi_manifests/registrar/manifest.json
@@ -0,0 +1,5 @@
+{
+    "name": "registrar",
+    "path": "/opt/smartdc/registrar/etc/config.json",
+    "master": true
+}
diff --git a/sapi_manifests/registrar/template b/sapi_manifests/registrar/template
new file mode 100644
index 0000000..8236209
--- /dev/null
+++ b/sapi_manifests/registrar/template
@@ -0,0 +1,30 @@
+{
+  "registration": {
+    "domain": "{{SERVICE_DOMAIN}}",
+    "type": "rr_host",
+    "service": {
+      "type": "service",
+      "service": {
+        "srvce": "_http",
+        "proto": "_tcp",
+        "port": 80
+      },
+      "ttl": 60
+    },
+    "ttl": 60
+  },
+
+  "zookeeper": {
+    "servers": [
+      {{#ZK_SERVERS}}
+        {
+          "host": "{{host}}",
+          "port": {{port}}
+        }{{^last}}, {{/last}}
+      {{/ZK_SERVERS}}
+    ],
+    "timeout": 60000
+  },
+
+  "adminIp": "{{auto.ADMIN_IP}}"
+}
